src/models.coffee

```diff
@@ -112,6 +112,22 @@ FormRenderer.Models.ResponseField = Backbone.DeepModel.extend
 
     prevValue != @isVisible
 
+  # Determines whether this field is hidden on a brand new response form.
+  # This is a helper method for FormBuilder.
+  isDefaultHidden: (fieldCollection) ->
+    if @get('admin_only') == true
+      true
+    else if @isConditional()
+      visible = _[@conditionMethod()] @getConditions(), (c) =>
+        (new FormRenderer.ConditionChecker(
+          null,
+          c,
```

Comment:
This is super-nitpicky, but can we use a better variable name than `c` here? I'm assuming it stands for condition, but with `@conditionMethod` and `@getConditions` it seems like an easy way to simplify the mental complexity here. I had to go check the `ConditionChecker` initializer to be sure.

Reply:
good call, will do